pandas/plotting/_core.py

```diff
@@ -2738,18 +2738,81 @@ def bar(self, x=None, y=None, **kwds):
 
     def barh(self, x=None, y=None, **kwds):
         """
-        Horizontal bar plot
+        Make a horizontal bar plot.
+
+        A horizontal bar plot is a plot that presents categorical data with
+        rectangular bars with lengths proportional to the values that they
+        represent. A bar plot shows comparisons among discrete categories. One
+        axis of the plot shows the specific categories being compared, and the
+        other axis represents a measured value.
 
         Parameters
         ----------
-        x, y : label or position, optional
-            Coordinates for each point.
-        `**kwds` : optional
+        x : label or position, optional
+            Column to be used for categories.
```

Comment:
Can you describe the defaults. For `x` I think it's the DataFrame's index. For `y` I think it's all numeric columns.